ui/browser/views/navbar/btn.jsx

```diff
@@ -12,21 +12,41 @@ specific language governing permissions and limitations under the License.
 
 import React, { PropTypes } from 'react';
 
+const HEIGHT = "20px";
+const WIDTH = "20px";
+
 /**
  * A button for the NavBar.
- * `icon` must be a valid font-awesome icon name
+ * `image` must be a valid CSS path to an image.
  */
-const Btn = ({ disabled, title, icon, onClick }) => (
-  <a href="#" className={disabled ? 'disabled' : ''} {...{ title, onClick }}>
-    <i className={`fa fa-${icon}`} />
+const Btn = ({ disabled, title, image, onClick }) => (
+  <a href="#"
+     style={{
+       backgroundImage: image ? `url(assets/${image})` : `url(${DEFAULT})`,
```

Comment:
@victorporof can explain the reasoning here better, this was more to try it out, I think he wants to discuss at tomorrows meeting